.github/workflows/e2e.yml

```diff
@@ -10,9 +10,12 @@ on:  # yamllint disable-line rule:truthy
       - Build
     branches:
       - main
+      - ISV-1504
```

Comment:
I'm assuming this is just for testing, and will be removed before merging, right?

Reply:
Yes, it is only for testing purposes. I will remove the config before merge.